actpool/actqueue_test.go

```diff
@@ -166,38 +167,6 @@ func TestActQueueAllActs(t *testing.T) {
 	require.Equal([]action.SealedEnvelope{tsf1, tsf3}, actions)
 }
 
-func TestActQueueRemoveActs(t *testing.T) {
```

Comment:
can you modify (or add) a test, to verify the actqueue's behavior is same before and after the change in this PR?